### ISSUE TITLE
[BUG] Missing visual indication for third-party notification, enemy down, etc

### ISSUE THREAD

**Issue by apex-legends-bugs** [apex-legends-bugs/bugs#44]
**Describe the bug**
When you get third partied, if you have voicelines turned on and have good enough hearing, you can hear a callout by one of the legends saying there's a third party happening. However, not everyone has good hearing, and some people who do turn off the voice lines because they overpower all the other sounds. There are several [barks](https://www.thenarrativedept.com/blog/barks) that are missing:

- Taking Fire
- Firing on enemy
- Downed an enemy
- Another enemy down
- Killed an enemy
- Whole squad down
- Another squad attacking
- Frag out

**To Reproduce**
1. Turn down voice line volume to 0
2. Play a game
3. You're missing a lot of context

**Expected behavior**
There should be an indication for those things in the lower left, right where you see that they're healing or getting res'd

- Taking Fire - a "fire explosion" icon, like this: 💥
- Firing on enemy - a gun icon
- Downed an enemy - a down arrow icon: ⬇
- Another enemy down - a double down arrow icon: ⬇⬇
- Downed multiple enemies - a down arrow with a plus: ⬇➕
- Killed an enemy - a skull icon: 💀
- Whole squad down - an icon with three skulls, kind of like the "Triple Triple" badge
- Another squad attacking - an icon with three silhouettes on it, like the "Master Of All" badge
- Frag out: grenade icon (change depending on grenade type)

Additionally, this new notification should be added:
- Cracked enemy shield - this should happen if a person shoots an enemy and cracks them. It should show a cracked shield icon and eventually should have associated barks added to the voice lines. This is a very important piece of information in competitive play and it is a big oversight not to include it in the voice lines.

**Why could this be happening?**
Respawn aren't great with accessibility

**System (please complete the following information):**
 - Platform: All
 - When exactly the issue happened (exact day and date): Always
